src/app/business/routes/model/components/list/components/popoverItems.js

```diff
@@ -1,19 +1,91 @@
 import React from 'react';
-import PopoverItems, {
-PopList, FilterUpPopItem, CopyPopItem,
+import PropTypes from 'prop-types';
+import {capitalize} from 'lodash';
+import {
+PopList, FilterUpPopItem, PopItem, Action,
 } from '../../../../../common/components/list/components/popoverItems';
+import {deepGet} from '../../../../../../utils/selector';
 
-
-const ModelPopoverItems = ({
-model, filterUp, addNotification,
-}) => (
-    <PopList>
-        <FilterUpPopItem filterUp={filterUp} />
-        <CopyPopItem model={model} addNotification={addNotification} />
-    </PopList>
+const CopyPopItem = ({addNotification, assetName, assetKey}) => (
+    <PopItem>
+        <Action onClick={addNotification(assetKey, `${capitalize(assetName)}'s key successfully copied to clipboard!`)}>
+            {`Copy ${assetName}'s key to clipboard`}
+        </Action>
+    </PopItem>
 );
 
-ModelPopoverItems.propTypes = PopoverItems.propTypes;
-ModelPopoverItems.defaultProps = PopoverItems.defaultProps;
+CopyPopItem.propTypes = {
+    addNotification: PropTypes.func.isRequired,
+    assetName: PropTypes.string.isRequired,
+    assetKey: PropTypes.string.isRequired,
+};
+
+
+const StandardModelPopoverItems = ({item, filterUp, addNotification}) => {
+    const modelKey = deepGet('traintuple.outModel.hash')(item);
+    const traintupleKey = deepGet('traintuple.key')(item);
```

Comment:
Shouldn't we put the selector calls in a `mapStateFromProps` function?

Reply:
In this case there's no mapStateFromProps, the item is passed by the parent component. Also, `deepGet` isn't really a selector, just a helper that is used by selectors.